transformers/events/cat_rely/converter_test.go

```diff
@@ -0,0 +1,47 @@
+package cat_rely_test
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/makerdao/vdb-mcd-transformers/test_config"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/events/cat_rely"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/test_data"
+	"github.com/makerdao/vulcanizedb/libraries/shared/factories/event"
+	"github.com/makerdao/vulcanizedb/pkg/core"
+	"github.com/makerdao/vulcanizedb/pkg/datastore/postgres"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Cat Rely converter", func() {
+	var (
+		converter event.Converter
+		db        *postgres.DB
```

Comment:
Probably won't really make a difference performance-wise, but do we want to initialize the `db` outside of the `BeforeEach` for consistency's sake?